docs/source/guide/s3.rst

```diff
@@ -497,10 +497,9 @@ To generate a pre-signed URL, use the
     # to send the GET, but we will use requests here to keep things simple.
     response = requests.get(url)
 
-If your bucket requires the use of signature version 4, you can elect to use it
-to sign your URL. This does not fundamentally change how you use generator,
-you only need to make sure that the client used has signature version 4
-configured.::
+The boto client will generate a sigv4 (s3v4) signature by default for new regions that 
```

Comment:
Can you change "The boto client" to either "The boto3 client" or the "S3 client".  I think we should be careful about using "boto", which usually refers to the older deprecated SDK.  Otherwise looks good, thanks for updating.